libs/ngx-dsfr/navigation/src/lib/testing/navigation.component.spec.ts

```diff
@@ -0,0 +1,217 @@
+/**
+ * Angular imports
+ */
+import { ComponentFixture, TestBed } from '@angular/core/testing';
+import { By } from '@angular/platform-browser';
+import { HarnessLoader } from '@angular/cdk/testing';
+import { TestbedHarnessEnvironment } from '@angular/cdk/testing/testbed';
+import { Router } from '@angular/router';
+
+/**
+ * 3rd-party imports
+ */
+import { RouterLinkDirectiveStub } from '@betagouv/ngx-dsfr/testing';
+
+/**
+ * Internal imports
+ */
+import {
+  DsfrNavigationComponent,
+  EMPTY_NAVIGATION_ERROR,
+  Navigation,
+} from '../navigation.component';
+import { TestHostComponent } from './test-host.component';
+import { DsfrNavigationHarness } from './navigation.harness';
+
+describe('DsfrNavigationComponent', () => {
+  let fixture: ComponentFixture<TestHostComponent>;
+  let testHost: TestHostComponent;
+  let componentUnderTest: DsfrNavigationComponent;
+  let harnessLoader: HarnessLoader;
+  let dsfrNavigationHarness: DsfrNavigationHarness;
+
+  const testNavigation: Navigation = [
+    {
+      "id": "a",
+      "label": "MEGA MENU",
+      "userRoles": [
+        "ROLE_USER",
+        "ROLE_ADMIN"
+      ],
+      "href": "modules",
+      "child": {
+        "isMega": true,
+        "title": "Titre éditorialisé",
+        "description": "Lorem [...] elit ut.",
+        "items": [
+          {
+            "id": "b",
+            "label": "Config",
+            "href": "admin/dashboard/config",
+            "categories": [
+              {
+                "id": 'c',
+                "label": "Config 1",
+                "href": "admin/dashboard/config",
+              },
+              {
+                "id": 'f',
+                "label": "Config 2",
+                "href": "/modules/navigation"
+              },
+            ]
+          },
+          {
+            "id": "b2",
+            "label": "Config",
+            "href": "admin/dashboard/config",
+            "categories": [
```

Comment:
It seems this object doesn't follow the new definition for the `NavigationItem` interface